### DISCUSSION TITLE
Unable to add key. apt-get(8) deprecated

### DISCUSSION THREAD

**Discussion by DScottF78** [abraunegg/onedrive#1904] [Q&A] [answered]
I was able to add onedrive, but unable to add the key for updates. This is the message...

```
Surface-Pro-4:~$ wget -qO - https://download.opensuse.org/repositories/home:/npreining:/debian-ubuntu-onedrive/xUbuntu_21.10/Release.key | sudo apt-key add -
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
OK
```

Is there an alternate method I can use to add the needed key?

**Comment by abraunegg** [comment, marked as answer]
@DScottF78 
The key is actually added - that is a warning message - this is why it returns OK

**Reply by DScottF78** [reply]
What a blistering fast reply. Thank for the you the clarification. It's working great for everything I've tried so far. Really good job.

**Reply by abraunegg** [reply]
@DScottF78 
FYI - the instructions are also now updated to avoid the depreciation warning

**Comment by DScottF78** [comment]
That's great. It really helps newcomers like me gain confidence when
working with the terminal.
